spec/decanter/parser/boolean_parser_spec.rb

```diff
@@ -61,5 +75,33 @@
           .to raise_error(Decanter::ParseError)
       end
     end
+
+    context 'returns true with options for' do
+      trues_with_options.each do |cond|
+        it "#{cond[0]}: #{cond[1]}, option: {#{cond[2]}: #{cond[3]}}" do
+          expect(parser.parse(name, cond[1], true_value: cond[3])).to match({name => true})
+        end
+      end
+    end
+
+    context 'returns false with options for' do
+      falses_with_options.each do |cond|
+        it "#{cond[0]}: #{cond[1]}, option: {#{cond[2]}: #{cond[3]}}" do
+          expect(parser.parse(name, cond[1], true_value: cond[3])).to match({name => false})
+        end
+      end
+    end
+
+    context 'with empty string and empty options' do
+      it 'returns nil' do
+        expect(parser.parse(name, '', true_value: '')).to match({name => nil})
+      end
+    end
```

Comment:
This is an interesting edge case. Would you not expect setting `true_value: ""` to return `true`?

Reply:
In the `normalize` method we return `nil` if `blank` 
```ruby
return if (value.nil? || value.blank?) # line 22
```

Reply:
this was part of the originally functionality

Reply:
agreed - this also related to the them of a level of opinion in the parser